change/@fluentui-react-breadcrumb-027d665b-7783-4af2-86c0-5cbf7eaa5ec4.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix(breadcrumb): moved sized icons to styling",
```

```suggestion
  "comment": "fix: moved sized icons to styling",
```

manual scope is not needed for change-file - the scope itself is the changefile name